lib/temporal/client.rb

```diff
@@ -133,6 +133,10 @@ def register_namespace(name, description = nil)
       connection.register_namespace(name: name, description: description)
     end
 
+    def describe_namespace(name)
```

Comment:
Can you please add a YARD comment similar to other methods in this file? It's the widest public-facing API, so would be good to keep it documented